src/SSVOpenHexagon/Components/CWall.cpp

```diff
@@ -55,23 +55,34 @@ void CWall::update(HexagonGame& mHexagonGame, ssvu::FT mFT)
 void CWall::moveTowardsCenter(
     HexagonGame& mHexagonGame, const sf::Vector2f& mCenterPos, ssvu::FT mFT)
 {
+    const float wallSpawnDist{mHexagonGame.getLevelStatus().wallSpawnDistance};
+    const float wallAccSpawnDist{mHexagonGame.getLevelStatus().wallAccSpawnDistance};
+
     const float radius{mHexagonGame.getRadius() * 0.65f};
+    const float outerBounds =
+        (wallSpawnDist == wallAccSpawnDist) ? wallSpawnDist * 0.5f : 800.f;
 
     int pointsOnCenter{0};
+    int pointsOutOfBounds{0};
     for(sf::Vector2f& vp : vertexPositions)
     {
         if(std::abs(vp.x - mCenterPos.x) < radius &&
             std::abs(vp.y - mCenterPos.y) < radius)
         {
             ++pointsOnCenter;
-        }
+        } 
         else
         {
+			if(std::abs(vp.x - mCenterPos.x) > outerBounds &&
+                std::abs(vp.y - mCenterPos.y) > outerBounds)
+			{
+				++pointsOutOfBounds;
+			}
```

Comment:
Why was this necessary? When do normal walls or accelerating walls end up out of bounds? Maybe a timer approach would be cheaper compared to an extra collision check per frame?